### PR TITLE
Add template 3dnodes to assets

### PR DESCRIPTION
**Pull request by lynochka**
Added notebook converted via https://github.com/mwouts/jupytext to .py for better version control. 
Could be converted back while templates being added to dshub.

jupytext --to notebook template_3dnodes_assets.py
jupytext --to py template_3dnodes_assets.ipynb

### CODE REVIEW

File: entity_matching/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+To run the notebook locally install poetry with https://github.com/python-poetry/poetry, then:
```
Comment (anderscognite) [inline entity_matching/README.md]: Let's skip the local development readme for now maybe? Instead, we should have a readme that explains what this is.

File: entity_matching/template_3dnodes_assets.py [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+# we provide functions to simplify the logics of the notebook
+from utils import chunk_create_rules_df, chunk_predict, get_matches_with_rules
+
+# we provide functions to install 3d-nodes, assets and asset-mappings
+from data_load_cdf import load_assets, load_threednodes, filter_df_threednodes, load_asset_mappings
+# -
+
+# # Initialize
+
+project = "akerbp"
```
Comment (anderscognite) [inline entity_matching/template_3dnodes_assets.py]: This seems way too specific for `akerbp`. I think we need a simpler example.

**Comment by lynochka** [comment]
> This seems way too complicated to be a tutorial template. Also, we shouldn't use poetry as people will not use poetry in dshub most likely. At least not yet.

All the steps are necessary to provide meaning results for 3D nodes contextualization. So we need all the steps.

File: entity_matching/template_3dnodes_assets.py (hunk 2 above)
Comment (lynochka) [reply entity_matching/template_3dnodes_assets.py]: This is the same for other projects I believe. What was to specific besides the name of the project?

**Comment by lynochka** [comment]
> This seems way too complicated to be a tutorial template. Also, we shouldn't use poetry as people will not use poetry in dshub most likely. At least not yet.

If the versions are similar in DS, poetry files will and not and should not be exposed there. we at least need to have versioning when we create a template and want to reproduce it.

**Comment by anderscognite** [comment]
This needs to be a notebook as people will run it as a notebook. We will preinstall the packages.

Can't have hard coded akerbp strings in python files etc, but in the notebook it could be ok. We need to simplify it too I think :)

File: entity_matching/template_3dnodes_assets.py (hunk 2 above)
Comment (lynochka) [reply entity_matching/template_3dnodes_assets.py]: Changed the project and key name.

**Pull request closed without merging**